<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewDb"/>
    <addaction name="actionConnectToDb"/>
    <addaction name="separator"/>
    <addaction name="actionCloseDb"/>
   </widget>
   <widget class="QMenu" name="menuEmployees">
    <property name="title">
     <string>Employees</string>
    </property>
    <addaction name="actionEmployeeAdd"/>
    <addaction name="separator"/>
    <addaction name="actionEmployeeEdit"/>
    <addaction name="actionEmployeeDelete"/>
    <addaction name="separator"/>
    <addaction name="actionEmployeesUpdate"/>
   </widget>
   <widget class="QMenu" name="menuTasks">
    <property name="title">
     <string>Tasks</string>
    </property>
    <addaction name="actionTaskAdd"/>
    <addaction name="actionTaskAddSub"/>
    <addaction name="separator"/>
    <addaction name="actionTaskEdit"/>
    <addaction name="actionTaskDelete"/>
    <addaction name="separator"/>
    <addaction name="actionTasksUpdate"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTasks"/>
   <addaction name="menuEmployees"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewDb">
   <property name="text">
    <string>New DB...</string>
   </property>
  </action>
  <action name="actionConnectToDb">
   <property name="text">
    <string>Connect To DB...</string>
   </property>
  </action>
  <action name="actionNewEmployee">
   <property name="text">
    <string>New Employee</string>
   </property>
  </action>
  <action name="actionDeleteEmployee">
   <property name="text">
    <string>Delete Employee</string>
   </property>
  </action>
  <action name="actionCreateTask">
   <property name="text">
    <string>Create Task ...</string>
   </property>
  </action>
  <action name="actionCloseDb">
   <property name="text">
    <string>Close DB</string>
   </property>
  </action>
  <action name="actionEmployeeAdd">
   <property name="text">
    <string>Add...</string>
   </property>
  </action>
  <action name="actionEmployeeEdit">
   <property name="text">
    <string>Edit...</string>
   </property>
  </action>
  <action name="actionEmployeeDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionEmployeesUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionTaskAdd">
   <property name="text">
    <string>Add...</string>
   </property>
  </action>
  <action name="actionTaskAddSub">
   <property name="text">
    <string>Add Subtask...</string>
   </property>
  </action>
  <action name="actionTaskEdit">
   <property name="text">
    <string>Edit...</string>
   </property>
  </action>
  <action name="actionTaskDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionTasksUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionFilters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
